### DISCUSSION TITLE
This issue is realted to PocketHost password reset

### DISCUSSION THREAD

**Discussion by ajayyy123** [pockethost/pockethost#233] [Q&A]
I have created instance using https://pockethost.io/app/instances/xycjuo16y2sdvns when user is trying to reset password through this 
code
--------------------------------------------------------------------------
const pb = new PocketBase("https://tart-flag.pockethost.io");
   pb.collection("users").requestPasswordReset(email).then((res) => {
        if (res) {
          alert("Reset link set on your Email address");
          goto("/login");
        }
      }); password link is sent on mail but we open its pointing to local host like this
----------------------------------------------------------------------------
 http://localhost:8090/_/#/auth/confirm-password-reset/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjb2xsZWN0aW9uSWQiOiJfcGJfdXNlcnNfYXV0aF8iLCJlbWFpbCI6ImRhbHdhbmkuYWpheUBnbWFpbC5jb20iLCJleHAiOjE2ODcyNTc1NDIsImlkIjoiZnVndWxld3pvNnVmazNzIiwidHlwZSI6ImF1dGhSZWNvcmQifQ.EiCfvn9qFq_39S4Ip8NCVNVKTYTg7Yqo8aJvT9e7_v4
-----------------------
Not sure I am doing right using pockethost instance pls guide

**Comment by benallfree** [comment]
@ajayyy123 You need to configure your email templates in the PocketBase admin settings https://tart-flag.pockethost.io/_/#/settings/mail

**Comment by ajayyy123** [comment]
its working now thanks...your product is super awesome .. well done.
